www/index.html

```diff
@@ -167,12 +167,13 @@
       <div id="footer">
         <div class="menu-group">
           <div class="menu-group-left"></div>
+          <div class="presenter-switch"> </div>
```

Comment:
should we remove the one in the settings eventually? doesn't have to be in this pr.

Reply:
Yeah need to discuss that with aman veerji. All these options are now at two places (specially display options).